index.html

```diff
@@ -2215,11 +2215,30 @@ <h3>Verification Relationship Validation</h3>
       <section>
         <h3>Canonicalization Method Correctness</h3>
 
-        <p class="issue">
-Canonicalization mechanisms utilized for normalizing input to hashing functions
-need to have vetted mathematical proofs associted with them. Canonicalization
-mechanisms that create collisions in hash functions can be used to attack
-digital signatures.
+        <p>
+One of the algorithmic processes that is used by this specification is
+canonicalization, which is a type of <a>transformation</a>. Canonicalization is
+the process of taking information that might be expressed in a variety of
+equivalent ways and expressing it in one way, called a "canonical form".
+        </p>
+        <p>
+The security of a resulting <a>data integrity proof</a> that utilizes
+canonicalization is highly dependent on the correctness of the algorithm. For
+example, if it is possible to confuse a canonicalization algorithm such that
+multiple different inputs produce the same output, then it's possible for one
```

Comment:
This needs some rephrasing. I may come up with better wording, but more minds thinking about it may bring better words faster.

Canonicalization of `multiple different input[ file]s` *should* `produce the same` _canonicalized_ `output`.

The concern is about the data within the inputs... Maybe `multiple different input data sets`?

This *might* be made clearer with an example, if a simply presentable one exists.

Reply:
I've been racking my brain to try and think of a simple example. To create a simple example, you have to provide and explain a canonicalization function, like one that always maps a value to the number `1`... and that example is so obviously wrong, that people will probably think it's contrived (and possibly miss the point). Going a bit more advanced, you can provide a canonicalization function c(n) where, if `n` contains the latter `a`, then the result is always the number `1` (which would demonstrate the collision)... but these examples are incredibly contrived and it's not clear to me if it helps or harms.

Your "multiple different input data sets" is an improvement, so I'm happy to apply it if we can't come up with something better.

Reply:
I think "confuse a canonicalization algorithm" is the most odd bit here. Perhaps:

```
For example, if a canonicalization algorithm converts two inputs that have a different meaning into the same output, then the author's intentions can be misunderstood by a verifier.
```

Reply:
Applied @dlongley's wording in 9044a8149605e09d64279d41979e4b9e0813d71a. Does that help @TallTed?